docs/node/run-your-node/ias-proxy.md

```diff
@@ -58,18 +58,13 @@ In order to configure the IAS proxy create the `/node/ias/config.yml` file with
 the following content:
 
 ```yaml
-datadir: /node/ias
-
-log:
-  level:
-    default: info
-  format: JSON
-
-ias:
-  production: true
-
-grpc:
-  port: 8650
+mode: client
+common:
+    data_dir: /node/ias
+    log:
+        format: JSON
+        level:
+            default: info
```

Comment:
I think that the migrate command doesn't support IAS node properly.

Reply:
Can you add the config example for the IAS node? I do not know how it looks.

Reply:
Config only has:
```yaml
common:
  data_dir: /srv/ias/data
  log:
    level:
      default: info
    format: JSON
```
And then is started as (note that these cannot be configured in the config file at the moment):
```sh
/srv/node/bin/oasis-node ias proxy \
  --config /srv/ias/etc/config.yml \
  --address unix:/srv/node/data/internal.sock \
  --ias.production true \
  --grpc.port 8650
```

Reply:
Added this to the docs